homeassistant/components/homekit_controller/__init__.py

```diff
@@ -25,6 +25,7 @@
     'lightbulb': 'light',
     'outlet': 'switch',
     'thermostat': 'climate',
+    '4aaaf930-0dec-11e5-b939-0800200c9a66': 'sensor'
```

Comment:
Ideally this would be added to the upstream homekit library rather than leaving it as a raw UUID here - the main concern is that if you leave it as the raw value and it later gets added, this will probably silently break.

Reply:
(Or is this UUID Kogeek-specific?)

Reply:
There is one issue with generic services and characteristics though in sensor and that is why I didn't enable it without proper testing. Which characteristic is the default one and what is its unit of measurement.